include/zephyr/net/wifi_mgmt.h

```diff
@@ -768,6 +776,124 @@ struct wifi_ap_config_params {
 	uint32_t max_num_sta;
 };
 
+/** @brief Wi-Fi DPP configuration parameter */
+/** Wi-Fi DPP QR-CODE in string max len for SHA512 */
+#define WIFI_DPP_QRCODE_MAX_LEN 255
+
+/** Wi-Fi DPP operations */
+enum wifi_dpp_op {
+	WIFI_DPP_OP_INVALID = 0,
+	WIFI_DPP_CONFIGURATOR_ADD,
+	WIFI_DPP_AUTH_INIT,
+	WIFI_DPP_QR_CODE,
+	WIFI_DPP_CHIRP,
+	WIFI_DPP_LISTEN,
+	WIFI_DPP_BOOTSTRAP_GEN,
+	WIFI_DPP_BOOTSTRAP_GET_URI,
+	WIFI_DPP_SET_CONF_PARAM,
+	WIFI_DPP_SET_WAIT_RESP_TIME
```

Comment:
Please document these operations. If the operations are more or less internal and not exposed to users or applications, then we could also mark them as internal with
```
/** @cond INTERNAL_HIDDEN */
...
/** @endcond */
```
which prevents the comments from generated documentation. See examples how this is used in this file for example.

Reply:
These enums are external. But I think these enum names already explains themselves.
I can add comments for them. But these comments will be similar to names. It that okay?